### DISCUSSION TITLE
Gamma pass rate

### DISCUSSION THREAD

**Discussion by LGDi** [e0404/matRad#580] [Q&A] [answered]
hello，
I got some results from running in topas, I used the code 'matRad_gammaIndex(cube1,cube2,resolution,criteria,slice,n,localglobal,cst)' to calculate the gamma pass rate between the topas results and the results obtained by direct simulation of matRad.But I found that they are only around 65%, why is it so low?
![image](https://user-images.githubusercontent.com/96415745/194706422-5b0a8022-74fb-46e7-9f7e-8f6d054386d4.png)

**Comment by wahln** [comment, marked as answer]
See my answer to #581 and the PR #536 , there will be quite an update which improves this. But generally, theres three things to consider:
- matRad's generic base data so far does not contain detailed information about beam optics or energy spectra. The Monte Carlo interface tries to approximate those, but it might not be accurate. In the above mentioned update, you will also be able to store gaussian energy spectra and bigaussian emittance parameterizations. 
- The gamma-index sometimes is not straightforward, because it strongly depends on size of the dose grid / voxel resolution in relation to the criterion together with the number of interpolation points used. You can use the function matRad_compareDose to also get profiles to compare easily.

**Comment by LGDi** [comment]
![image](https://user-images.githubusercontent.com/96415745/199969053-0b1204c0-db1d-4bf5-841a-7768839fe44a.png)
This graph is the graph obtained from the Topas simulation results. But its overall dose is so uneven.

**Comment by wahln** [comment]
Definitely not enough histories used for the simulation.

**Reply by wahln** [reply]
And there seems to be something off in the normalization

**Reply by LGDi** [reply]
Yes, there are these two problems now, and the normalization has not been carried out. Is there any function in matrad that can complete the normalization.

**Reply by wahln** [reply]
How did you start the simulation?
If you run matRad_calcDoseDirectMC with TOPAS as engine  (`pln.propMC.proton_engine = 'TOPAS'`) it should automatically import and normalize. Otherwise, you need to calculate the normalization yourself from the number of histories & batches used. The #536 will also add some functionality here, i.e., automatically importing this after sending the parameter files to, e.g., an external cluster or something.

**Reply by LGDi** [reply]
Thank you for your reply! It seems that I need to calculate the normalization myself.
